packages/skill-api/src/core/services/process-stripe-webhook.ts

```diff
@@ -129,10 +134,22 @@ export async function processStripeWebhooks({
         }
       }
     } catch (err: any) {
-      console.error(err)
-      return {
-        status: 400,
-        body: `Webhook Error: ${err.message}`,
+      // To avoid making too much noise in Stripe, we are going to report a
+      // instance of No Associated Product to Sentry and then return a 200
+      // response from here.
```

Comment:
this comment is a lie 🍰

Reply:
@joelhooks yeah, I originally thought the issue was just creating noise in stripe, but the truth is that too high of an error rate and Stripe kills your webhook, right?

Reply:
I meant the Sentry bit because of the comment and was making a Portal joke 🤡

Reply:
whoops, right over my head. Portal joke now received and appreciated 🙌